grype/matcher/common/cpe_matchers.go

```diff
@@ -58,8 +58,9 @@ func FindMatchesByPackageCPE(store vulnerability.ProviderByCPE, p pkg.Package, u
 						"cpe": cpe.BindToFmtString(),
 					},
 					SearchMatches: map[string]interface{}{
-						"cpes":       cpesToString(vuln.CPEs),
-						"constraint": vuln.Constraint.String(),
+						"grypeDbNamespace":  vuln.Namespace,
```

Comment:
technically this is not a namespace from grype DB though right? Can it be `Namespace` instead? or what is the benefit of mentioning grype-db?

Reply:
I originally was going to start with `namespace` then realized that it's not clear what this is referring to... so I clarified it by adding `grypeDb` in front. I'm alright with `namespace`? I  didn't feel very strongly either way.